src/libraries/Common/src/System/Security/Cryptography/ECDsaOpenSsl.cs

```diff
@@ -15,7 +15,7 @@ public sealed partial class ECDsaOpenSsl : ECDsa, IRuntimeAlgorithm
         // secp521r1 maxes out at 139 bytes, so 256 should always be enough
         private const int SignatureStackBufSize = 256;
 
-        private ECOpenSsl? _key;
+        private Lazy<SafeEvpPKeyHandle>? _key;
 
```

Comment:
I think I'd be happier if ECDSA and ECDH were still in terms of ECOpenSsl than going back to having a lot of simple-looking-but-important code duplicated across them.  But I'm OK with saying they can be re-unified in the indefinite future when we can stop talking about EC_KEY (except for the ctors that take them... though we could decide to Obsolete + PNSE them... which would let EC_KEY disappear from the managed side altogether)